airflow/providers/influxdb/example_dags/example_influxdb.py

```diff
@@ -0,0 +1,57 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
please add `__init__` to `example_dags`

Reply:
Thanks @eladkal , added and rebased again, fingers crossed.